data/README.md

```diff
@@ -1,16 +1,23 @@
 Data
 ----
 
+Training vs test data
+---------------------
+
+We train on all data, without labels. We use the labels in order to evaluate the resulting clusters.
+
 Twitter Airlines Customer Support
 ---------------------------------
 
 The data is available in two version:
 
-- raw: minimal redaction (company and customer twitter id), no preprocessing: [airlines_raw_merged.csv.bz2](airlines_raw_merged.csv.bz2)
-- redacted, and preprocessed:  [airlines_500_mergedb.csv.bz2](airlines_500_mergedb.csv.bz2)
+- raw: minimal redaction (company and customer twitter id), no preprocessing: [airlines_raw.csv.bz2](airlines_raw.csv.bz2)
+- redacted, and preprocessed:  [airlines_processed.csv.bz2](airlines_processed.csv.bz2)
+
+492 examples are labeled using annotators. The remaining examples are labeled `UNK`.
```

Comment:
Can you say that we sampled 500 examples and only labeled 492 due to others are written in other languages or contain swear words or something?

Reply:
ok. addressed in 
593ed46